contracts-periphery/Makefile

```diff
@@ -12,11 +12,11 @@ forkBlock = 14800081
 build :; forge build
 clean :; forge clean
 install :; $(INSTALL_CMD)
-snapshot-gas :; forge test -f ${ETH_RPC_URL} --fork-block-number ${forkBlock} --match-path *.gas.t.sol --gas-report > snapshot/.gas
+snapshot-gas :; forge test --match-path *.gas.t.sol --gas-report > snapshot/.gas
 test :; make test-contracts test-gas
 test-contracts :; make test-batch-send test-uniswap-hook
 test-batch-send :; forge test --match-path test/UmbraBatchSend.t.sol
-test-uniswap-hook :; forge test -f ${ETH_RPC_URL} --fork-block-number ${forkBlock} --match-path test/UniswapWithdrawHook.fork.1.t.sol
+test-uniswap-hook :; forge test --match-path test/UniswapWithdrawHook.fork.1.t.sol
 test-gas :; make test-gas-batch-send test-gas-uniswap-hook
 test-gas-batch-send :; forge test --match-path test/gas/UmbraBatchSend.gas.t.sol
-test-gas-uniswap-hook :; forge test -f ${ETH_RPC_URL} --fork-block-number ${forkBlock} --match-path test/gas/UniswapWithdrawHook.fork.1.gas.t.sol
\ No newline at end of file
+test-gas-uniswap-hook :; forge test --match-path test/gas/UniswapWithdrawHook.fork.1.gas.t.sol
```

Comment:
Should we clean up this makefile a bit? We probably just need `make test` and `make test-gas` (and `make snapshot-gas`) now, since the new fork architecture means any specific test file can just be run with the standard `forge test --match-path` flag so we prob don't need special commands in the makefile for each

Reply:
It's def cleeean https://github.com/ScopeLift/umbra-protocol/blob/6f3caa0080de5df43fd12b1afc35f2444539df69/contracts-periphery/Makefile#L15-L17

Reply:
Love it 😍